generator/server_test.go

```diff
@@ -149,7 +149,6 @@ func TestServer_MultipartForm(t *testing.T) {
 func TestServer_InvalidSpec(t *testing.T) {
 	opts := testGenOpts()
 	opts.Spec = "../fixtures/bugs/825/swagger.yml"
-	opts.ValidateSpec = true
```

Comment:
This has caused the CI Appveyor to hang I believe.
We should not let the generation run on this spec : the Validate spec is I believe important